llvm/test/CodeGen/AArch64/neon-partial-reduce-dot-product.ll

```diff
@@ -948,3 +1295,5 @@ end:
   %2 = add <4 x i32> %psum2, %psum1
   ret <4 x i32> %2
 }
+;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
+; CHECK-I8MM: {{.*}}
```

Comment:
The note is still here which means that CHECK-I8MM is not used anywhere. My suggestion would be to keep CHECK-I8MM in the RUN line, and use CHECK-NEWLOWERING-I8MM for the new RUN line you added.